engine/multisig/src/crypto.rs

```diff
@@ -35,7 +35,7 @@ pub const MAX_SCALAR_SIZE: usize = 32;
 /// Used as a unique identifier when serializing/deserializing chain specific data.
 /// The values are explicitly given and should never be changed.
 #[repr(u16)]
-#[derive(Clone, Copy, Debug, FromPrimitive)]
+#[derive(Clone, Copy, Debug, FromPrimitive, PartialEq, Eq, PartialOrd, Ord)]
```

Comment:
What is this for? Particularly PartialOrd and Ord seem undesirable to impl.

Reply:
Left over from some experimenting that i did, forgot that i didn't need it anymore.